Mathlib/RingTheory/Ideal/Quotient.lean

```diff
@@ -299,6 +299,7 @@ section Pi
 
 variable (ι : Type v)
 
+set_option maxHeartbeats 300000 in
```

Comment:
I think this is now needed due to changes in the import graph

Reply:
Hmm, after much investigation the new import graph only imports two additional files,
* `Mathlib/Order/Hom/Bounded.lean`
* `Mathlib/Order/Hom/Lattice.lean`